src/backend/src/prisma/migrations/20231023224608_make_pdm_filename_optional_for_assembly/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "Assembly" ALTER COLUMN "pdmFileName" DROP NOT NULL;
```

Comment:
you should delete the old bom migration and just make a new one so it has all the bom stuff